.github/workflows/test.yml

```diff
@@ -0,0 +1,103 @@
+name: Tests
+on:
+  push:
```

Comment:
Can we add a `pull_request` trigger here as well to have the checks show up in PRs.